resources/views/includes/structure/related-pages.blade.php

```diff
@@ -1,12 +1,27 @@
 @if(!empty($records))
-    @section('associated_pages')
+@section('associated_pages')
+<div class="container">
+    <h3>Related to this page</h3>
```

Comment:
Hi @ostafinskim ,

It looks like you have duplicated the container element and title. Could you please remove the duplicates?

Reply:
Done it.